src/main/java/nl/tudelft/ti2206/group9/gui/MainMenuScene.java

```diff
@@ -98,4 +109,51 @@ public void handle(final ActionEvent event) {
 		});
 	}
 
+	/**
+	 * Checks whether the playername is a valid name.
+	 * Invalid options:
+	 * <a>
+	 *     <li>Empty name</li>
+	 *     <li>Name contains a '.'</li>
+	 *     <li>Name contains a '/'</li>
+	 *     <li>Name contains a '\'</li>
+	 * </a>
+	 * @param name the name to be checked
+	 * @return boolean to indicate whether the name is valid
+	 */
+	private boolean checkPlayerName(final String name) {
+		return !(name.contains(".") | name.contains("/") | name.contains("\\"));
```

Comment:
Findbugs doesn't really like this.
I quote: "Potentially dangerous use of non-short-circuit logic in nl.tudelft.ti2206.group9.gui.MainMenuScene.checkPlayerName(String)"

<review,bug